arbitrum-docs/node-running/how-tos/running-an-orbit-node.mdx

```diff
@@ -12,65 +12,117 @@ This how-to provides step-by-step instructions for running an Orbit node on your
+    - When using the flag `--chain.info-json=<Orbit Chain's chain info>`, it is essential to replace `<Orbit Chain's chain info>` with the specific `chain info` JSON string of the Orbit chain for which you wish to run the node: 
+
+- Example:
 
+  ```bash wordWrap=true
+  --chain.info-json="[{\"chain-id\":94692861356,\"parent-chain-id\":421614,\"chain-name\":\"My Arbitrum L3 Chain\",\"chain-config\":{\"chainId\":94692861356,\"homesteadBlock\":0,\"daoForkBlock\":null,\"daoForkSupport\":true,\"eip150Block\":0,\"eip150Hash\":\"0x0000000000000000000000000000000000000000000000000000000000000000\",\"eip155Block\":0,\"eip158Block\":0,\"byzantiumBlock\":0,\"constantinopleBlock\":0,\"petersburgBlock\":0,\"istanbulBlock\":0,\"muirGlacierBlock\":0,\"berlinBlock\":0,\"londonBlock\":0,\"clique\":{\"period\":0,\"epoch\":0},\"arbitrum\":{\"EnableArbOS\":true,\"AllowDebugPrecompiles\":false,\"DataAvailabilityCommittee\":false,\"InitialArbOSVersion\":10,\"InitialChainOwner\":\"0xAde4000C87923244f0e95b41f0e45aa3C02f1Bb2\",\"GenesisBlockNum\":0}},\"rollup\":{\"bridge\":\"0xde835286442c6446E36992c036EFe261AcD87F6d\",\"inbox\":\"0x0592d3861Ea929B5d108d915c36f64EE69418049\",\"sequencer-inbox\":\"0xf9d77199288f00440Ed0f494Adc0005f362c17b1\",\"rollup\":\"0xF5A42aDA664E7c2dFE9DDa4459B927261BF90E09\",\"validator-utils\":\"0xB11EB62DD2B352886A4530A9106fE427844D515f\",\"validator-wallet-creator\":\"0xEb9885B6c0e117D339F47585cC06a2765AaE2E0b\",\"deployed-at\":1764099}}]"
   ```
 
-  - Note for Anytrust chains, it is essential to add `--node.data-availability.enable`, and `--node.data-availability.rest-aggregator.urls=<A list of DAS REST endpoints>` or `--node.data-availability.rest-aggregator.online-url-list=<A url that returns a list of the DAS REST endpoints>` to the command or configuration.
+  - Please note: for Anytrust chains, it is essential to add the following flags to the command or configuration:
```

Comment:
This could be moved to the section above, since it's not actually an example

Reply:
Could you specify what isn't an example?

Reply:
I don't remember how this looked like before (there was an explanation of the Anytrust parameters in the "examples" section), but it's now good 💪